saas/office365/security_alerts/o365-security-alerts.pt

```diff
@@ -0,0 +1,144 @@
+name "Office 365 Security Alerts"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will identify Security Alerts that have been raised in Office 365. Policy Managers can minimize the notifications by choosing to only be alerted by certain severity level(s). See the [README](https://github.com/rightscale/policy_templates/tree/master/fsm/office_365/security_alerts/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "SaaS Management"
+
+parameter "param_azure_tenant_id" do
+  type "string"
+  label "Azure AD Tenant ID"
+  description "This value needs to be your Azure tenant ID.  \n To get your tenant ID see the article https://docs.microsoft.com/en-us/onedrive/find-your-office-365-tenant-id"
+  category "Azure"
+end
+
+parameter "param_alert_severity" do
+  label "Alert Severity"
+  type "string"
+  description "Specify the alert severity levels that should raise an incident"
+  allowed_values "Low,Medium,High","Medium,High","High"
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+auth "azure_auth", type: "oauth2" do
+  token_url join(["https://login.microsoftonline.com/",$param_azure_tenant_id,"/oauth2/v2.0/token"])
+  grant type: "client_credentials" do
+    client_id cred("O365_AAD_APPLICATION_ID")
+    client_secret cred("O365_AAD_APPLICATION_KEY")
+    additional_params do {
+      "scope" => "https://graph.microsoft.com/.default"
```

Comment:
Danger is complaining about this.  the host should be added to the exclude_hosts array in the Dangerfile.  Can yo include that in your PR.